vignettes/deployment.Rmd

```diff
@@ -92,17 +94,17 @@ Let's break down what happens here:
     
 ### Deploying to docs/ (master) or gh-pages branch
 
-#### master branch
+#### master branch deployment
 
 By default the deployment is done to the `docs/` directory of the `master` branch.
```

Comment:
Deploying to `docs/` leads to nasty merge conflicts with long-living pull requests. This is a strong enough reason to deploy to `gh-pages` by default -- I already changed the code in `do_pkgdown()`.